### DISCUSSION TITLE
Workers

### DISCUSSION THREAD

**Discussion by diegolr62** [postalserver/postal#2037] [Help with using Postal] [answered]
I need to increase workers for : smtp, rabbitmq. In version 1 there was an example postal.yml showing those settings. Please point me to the documentation or example to find those parameters in version 2.x

DiegoL

**Comment by diegolr62** [comment, marked as answer]
Closed, I've found the example in the code

**Reply by willpower232** [reply]
also have a look at the scale option which is here somewhere
